pkg/libvmi/vm.go

```diff
@@ -65,6 +65,12 @@ func WithRunning() VMOption {
 	}
 }
 
+func WithRunStrategy(strategy v1.VirtualMachineRunStrategy) VMOption {
+	return func(vm *v1.VirtualMachine) {
+		vm.Spec.RunStrategy = &strategy
```

Comment:
I'd expect here to have:
```
vm.Spec.Running = nil
```

Reply:
Makes sense, but requires `vm.Spec.RunStrategy =  nil` after line 64 for symmetry.